assignments/prototypes.js

```diff
@@ -15,14 +15,37 @@
   * destroy() // prototype method -> returns the string: 'Object was removed from the game.'
 */
 
-/*
+function GameObject(gameObjectAttributes) {
+	this.createdAt = gameObjectAttributes.createdAt;
+	this.dimentions = gameObjectAttributes.dimentions;
+}
+
+GameObject.prototype.destroy = function() {
+	return 'Object was removed from the game.';
+}
```

Comment:
This method is actually supposed to use the GameObject's name, not the word Object. So `this.name` instead, or something similar.